robustirc-bridge/bridge.go

```diff
@@ -41,6 +48,41 @@ var (
 // - for resuming sessions (later): the last seen message id, perhaps setup messages (JOINs, MODEs, …)
 // for hosted mode, this state is stored per-nickname, ideally encrypted with password
 
+func prefixMotd(msg string) string {
+	// The user chose to not inject a MOTD.
+	if *motdPath == "" {
+		return msg
+	}
+
+	sep := strings.Index(msg[1:], ":")
+	if sep == -1 {
+		return msg
+	}
+
+	prefix := msg[:sep+2] + "- "
+
+	f, err := os.Open(*motdPath)
```

Comment:
Can we put that in init() or main() and keep the MOTD in memory? should be used fairly often and no use adding a potential failure case here, just to save a couple of hundred bytes of RAM, imho.